src/main/java/com/example/dog/dao/InMemoryDogDAO.java

```diff
@@ -28,9 +28,8 @@ public Dog update(int id, Dog dog) {
         Dog newValue = dogs.computeIfPresent(id, (k, v)-> dog);
```

Comment:
I think this line was more awkward. This would look simpler:

```
    public Dog update(int id, Dog dog) {
        if (!dogs.containsKey(id))
            return null;
        dog.setId(id);
        dogs.put(id, dog);
        return dog;
    }
```

Reply:
That's interesting...it was your idea to use `computeIfPresent(id, (k, v)-> dog)` method and I don't understand why this line now looks like awkward

Reply:
Just more complicated than it should be